src/fabric.ts

```diff
@@ -195,6 +195,38 @@ export class Fabric {
     );
   }
 
+  getGeoFabricMetaData() {
+    return this._connection.request(
+      {
+        method: "GET",
+        path: `/database/metadata`,
```

Comment:
not everywhere we require template literals.

Reply:
updated